### PR TITLE
Teambuilder: Filter Wormadam correctly in STABmons

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: js/search.js [hunk 1]
```diff
@@ -995,6 +995,8 @@
 					var baseTemplate = Tools.getTemplate(template.baseSpecies);
 					for (var j = 0; j < template.types.length; j++) {
 						if (template.battleOnly) continue;
+						// Temporary until all in-battle formes are supported
+						if (template.forme === 'Ultra' || template.forme === 'Pirouette' || template.forme === 'Zen' || template.baseSpecies === 'Castform') continue;
```
Comment (Zarel) [inline js/search.js]: ...?

Where is `template.battleOnly` set?

Comment (AustinXII) [reply js/search.js]: Looks like it’s one line above

Comment (KrisXV) [reply js/search.js]: BattleOnly currently only covers mega evolutions and primals

Comment (Zarel) [reply js/search.js]: @AustinXII I was asking for where it's set, not where it's read. :|

@KrisXV Sounds like that should be fixed.

**Comment by KrisXV** [comment]
I'll get to this in a bit, I'm having issues with my local file right now so it could take a while

**Pull request merged**